src/components/views/SupportedBy/SupportedBy.module.css

```
@@ -0,0 +1,70 @@
+
+    }
+
+    .title {
+        & h1 {
+            font-family: "Cal Sans";
+            font-weight: 600;
+            font-size: 1.5rem;
+            letter-spacing: 0.05em;
+            color: var(--fg);
+            opacity: 0.5;
+            /*color: transparent;
```

Stylelint (17.14.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)